pipelines/pipelines/cli/build-branch.yaml

```diff
@@ -478,18 +485,64 @@ spec:
       value: 
       - pipeline
       - start
-      - cli-snapshot-main-to-prod
+      - branch-isolated
       - -n
       - galasa-build
       - --prefix-name 
-      - cli-snapshot
+      - trigger-cli-main
```

Comment:
I think you meant to call this 'trigger-isolated-main'? This will be the prefix of the name of the generated pipelinerun so should say isolated here.